assets/css/_cutout_overlay.scss

```
@@ -0,0 +1,44 @@
+.c-overlay-cutout {
+  z-index: 800; // Below controls, above map
+  position: absolute;
+  inset: 0px;
+
+  &,
+  &__rect,
+  &__mask-rect {
+    width: 100%;
+    height: 100%;
+  }
+
+  --circle-radius: 0;
+
+  // Cannot check hover on _this_ component due to `pointer-events: none`
```

I feel like `this` is not clear enough.
Maybe somehow explicitly state what `this` is?

```suggestion
  // Cannot check hover on _this_(&) component due to `pointer-events: none`
```
```suggestion
  // Cannot check hover on the `&` component due to `pointer-events: none`
```
```suggestion
  // Cannot check hover on the `c-overlay-cutout` component due to `pointer-events: none`
```

---

I like the last of those three suggestions the best, personally.